code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -180,3 +189,219 @@
 	icon = 'icons/obj/eldritch.dmi'
 	icon_state = "eldrich_flask"
 	list_reagents = list(/datum/reagent/eldritch = 50)
+
+/obj/item/clothing/head/hooded/cult_hoodie/void
+	name = "void hood"
+	icon_state = "void_cloak"
+	flags_inv = NONE
+	flags_cover = NONE
+	desc = "Black like tar, doesn't reflect any light. Runic symbols line the outside, with each flash you loose comprehension of what you are seeing."
+	item_flags = EXAMINE_SKIP
+	armor = list(MELEE = 30, BULLET = 30, LASER = 30,ENERGY = 30, BOMB = 15, BIO = 0, RAD = 0, FIRE = 0, ACID = 0)
```

Comment:
I'm not entirely sure how interesting invisible armor might be for the game.
It's something that should require some discussing.

Reply:
well it gives you a very interesting choice, would you rather have stealth armor that doesn't protect you much or a very good armor that screams HERETIC

Reply:
Latter, personally. Being armored should be a choice to forego stealth.

Reply:
well, it is a personal choice afterall, some people would love to go stealth and it gives them an option to help out with that.

Reply:
We've removed stun gloves and other similar items for the same reason, and it has been one of the main concerns about sleeping carp. I don't think this is a direction we want to go in.

Reply:
well it serves the same purpose as the hidden compartment implant but is worse in every way first of all it takes your suit slot so you cannot wear any real armor while wearing this, you can only fit small items and specific heretic items inside of this, it can be removed by anyone. It is situational at best. Besides this in itself doesn't provide any real advantage in combat, since you cannot hide medium and big things in this, meaning most weapons just simply dont fit in it.

Reply:
Resolved over on discord